### ISSUE TITLE
Wafw00f fails to execute on multiple sites

### ISSUE THREAD

**Issue by gh0stwr1ter** [EnableSecurity/wafw00f#34]
Getting the following output when running the tool against different sites.

> root@kali:~# wafw00f -a https://google.com -v
> Traceback (most recent call last):
>   File "/usr/local/bin/wafw00f", line 4, in <module>
>     __import__('pkg_resources').run_script('wafw00f==0.9.4', 'wafw00f')
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 739, in run_script
>     self.require(requires)[0].run_script(script_name, ns)
>   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1494, in run_script
>     exec(code, namespace, namespace)
>   File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/EGG-INFO/scripts/wafw00f", line 4, in <module>
>     from wafw00f import main
>   File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/wafw00f/main.py", line 53, in <module>
>     from wafw00f.lib.evillib import oururlparse, scrambledheader, waftoolsengine
>   File "/usr/local/lib/python2.7/dist-packages/wafw00f-0.9.4-py2.7.egg/wafw00f/lib/evillib.py", line 14, in <module>
>     from bs4 import BeautifulSoup
>   File "build/bdist.linux-x86_64/egg/bs4/__init__.py", line 30, in <module>
>   File "build/bdist.linux-x86_64/egg/bs4/builder/__init__.py", line 314, in <module>
>   File "build/bdist.linux-x86_64/egg/bs4/builder/_html5lib.py", line 70, in <module>
> AttributeError: 'module' object has no attribute '_base'
>

**Comment by sandrogauci** [comment]
I am not getting this output on my machine. The problem might be related to beautiful soup on Kali. Will test and revert back

**Comment by sandrogauci** [comment]
I made use of the latest Kali live CD and also could not reproduce your problem. This does not appear to be a wafw00f issue or a Kali linux issue (which is beyond our control anyway). Having said that, a quick google search shows that this is related to beautifulsoup. See [this](http://stackoverflow.com/questions/38447738/ddg#38799613). Solution they say, is to upgrade your beautifulsoup4 and html5lib as follows:

```
pip install --upgrade beautifulsoup4
pip install --upgrade html5lib
```

**Comment by sandrogauci** [comment]
fixed #38
